sdk/python/lib/pulumi/provider/server.py

```diff
@@ -17,31 +17,30 @@
 
 """
 
-from typing import Dict, List, Set, Optional, TypeVar, Any, cast
 import argparse
 import asyncio
 import sys
+from typing import Any, Dict, List, Optional, Set, TypeVar, cast
 
 import grpc
 import grpc.aio
-
+import pulumi.resource
+import pulumi.runtime.config
+import pulumi.runtime.settings
```

Comment:
This looks like something in your local env changed to cause your tooling to believe pulumi is a 3rd party library rather than a local dependency.

Reply:
Yes, vscode seems really keen to reorder everything